openmc/mgxs/mgxs.py

```diff
@@ -212,11 +229,70 @@ def num_groups(self):
         return self.energy_groups.num_groups
 
     @property
+    def scores(self):
+        return ['flux', self.rxn_type]
+
+    @property
+    def filters(self):
+        group_edges = self.energy_groups.group_edges
+        energy_filter = openmc.Filter('energy', group_edges)
```

Comment:
Not sure if it would ever really be an issue, but looking at this property multiple times will result in `Filter`s that have different `id`s